### DISCUSSION TITLE
Webshare activity meter blocks accounts.

### DISCUSSION THREAD

**Discussion by valeriopet** [KevinBytesTheDust/CVAmp#263] [Q&A] [answered]
webshare blocked 2 subscriptions. is there an alternative to webshare?
![webshare](https://github.com/jlplenio/crude-twitch-viewer-bot/assets/85315252/728116ec-d544-4ae1-a09d-e08ebfdfadc8)

**Comment by jlplenio** [comment]
Discussion thread here: https://github.com/jlplenio/crude-twitch-viewer-bot/issues/262#issuecomment-2085164529
They seem to have added an activity meter to track proxy usage. I reached out to them for comment.

**Reply by valeriopet** [reply]
we have an alternative to webshare?

**Comment by Alexistexas9697** [comment]
yes i have same problem in webshare :/

**Comment by Alexistexas9697** [comment]
I think I have solved this problem for now by verifying my webshare account.

**Comment by jlplenio** [comment, marked as answer]
I raised the issue and concerns with webshare. They communicated to me, that they will significantly increase the limits of the activity meter.
Therefore, this should not trigger normal use with my tool anymore. 🎉
